docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>Run-on sentence. Suggestion:</p><p> </p><p>We can have a range of variables with different shapes and dimensions,. As long as we are careful that the facetting and mapping arguments do not conflict, the underlying <code>PlotCollection</code> used by <code>plot_dict</code> can combine all the different variables into a single plot. The provided aesthetics will be mapped only when relevant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>